### ISSUE TITLE
DAC_Info_And_Relationships_05 – Code sections not marked up programmatically

### ISSUE THREAD

**Issue by markrowen** [digital-land/digital-land#703]
- **WCAG Reference:** 1.3.1 Info and Relationships (Level A)
- **Page/URL:** https://www.planning.data.gov.uk/docs

<img width="298" height="292" alt="Image" src="https://github.com/user-attachments/assets/971c35a2-78e5-424c-8396-c79ea6316ae4" />

- **Description:**
  - Code sections on the page are presented in generic `<div>` elements and are not marked up programmatically using `<pre>` and `<code>` tags.
  - This affects all blocks of code on the page.
  - Example of current code:
    ```html
    <div class="app-code-block govuk-!-margin-bottom-4" tabindex="0">
      GET /entity.json?<br>
      latitude=51.5074&amp;<br>
      longitude=-0.1278&amp;<br>
      dataset=conservation-area&amp;<br>
      dataset=listedbuilding&amp;<br>
      dataset=green-belt&amp;<br>
      dataset=flood-riskzone&amp;<br>
      dataset=article-4-direction-area&amp;<br>
      dataset=tree-preservationzone&amp;<br>
      limit=100
    </div>
    ```

- **User Impact:**
  - Screen reader users cannot identify code sections correctly, making it difficult to distinguish code from regular content.

- **Solution/Recommendation:**
  - Ensure all code sections are marked up programmatically using `<pre>` and `<code>` tags.
  - Example:
    ```html
    <pre><code>
    GET /entity.json?
    latitude=51.5074&amp;
    longitude=-0.1278&amp;
    dataset=conservation-area&amp;
    dataset=listedbuilding&amp;
    dataset=green-belt&amp;
    dataset=flood-riskzone&amp;
    dataset=article-4-direction-area&amp;
    dataset=tree-preservationzone&amp;
    limit=100
    </code></pre>
    ```
  - Further information: https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/code
